index.html

```diff
@@ -1,19 +1,85 @@
-
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Karma</title>
+        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    </head>
+    <body>
+        <header class="header">
+            <img src="./img/karma-logo.svg" alt="Karma logo" class="logoImg"/>
+            <nav>
+                <ui class="navList">
+                    <li class="navItem">
+                        <a href="#" class="navLink"><span class="makeBlack">Meet Kamra</span></a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">How It works</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">store</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">Blog</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">Help</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">Login</a>
+                    </li>
+                </ui>
+            </nav>
+            <img class="hamburgerMenu" src="./img/menu-hamburger.svg" alt="menu bar"/>
+        </header>
+        <main>
+            <section class="introduction">
+                <h1>Introducing Karma</h1>
+                <p>Bring WiFi with you,everywhere you go.</p>
+                <button class="learnmoreBtn">learn more</button>
+            </section>
+            <section>
+                <div class="advantageTitle">
+                    <p>Everyone needs a little Karma.</p>
+                </div>
+                
+                <div class="advantage">
+                    <div class="advantageColumn">
+                        <img src="./img/icon-devices.svg" alt="devices picture">
+                        <p>Internet for all devices</p>
+                    </div>
+                    <div class="advantageColumn">
+                        <img src="./img/icon-coffee.svg" alt="coffee picture">
+                        <p>Boosts your productivity</p>
+                    </div>
+                    <div class="advantageColumn">
+                        <img src="./img/icon-refill.svg" alt="refill picture"> 
+                        <p>Pay as You Go</p> 
+                    </div>
+                </div>    
+            </section>
+            <section class="getKarma">
+                <div class="getkarmaBackground">
+                </div>
+                <div class="getkarmaTitle">
```

Comment:
Is there a better tag you can use to style a title rather than a combination of a `<div>` and `<p>`?

Reply:
can I use <section>? @Sam-Martin